airflow-core/docs/core-concepts/auth-manager/index.rst

```diff
@@ -172,6 +172,40 @@ cookie named ``_token`` before redirecting to the Airflow UI. The Airflow UI wil
 .. note::
     Do not set the cookie parameter ``httponly`` to ``True``. Airflow UI needs to access the JWT token from the cookie.
 
+Refreshing JWT Token
+''''''''''''''''''''
+The refresh token ``<METHOD> <PATH>`` is ``POST /auth/token/refresh``. It returns a new JWT token in the cookie where updated token is stored.
+It requires the user to be authenticated, and it is usually called by the Airflow UI/API when the JWT token is about to expire.
+This endpoint is used to refresh the JWT token when it is about to expire.
+The auth manager should implement this endpoint to allow the Airflow UI/API to refresh the JWT token.
```

Comment:
We should also mention why an auth manager should override this method. If the auth manager does not use any underlying token underneath, there is no value implementing this method. An auth manager should override this method if it uses a token which expires and need to be refreshed (like Keycloak)

Reply:
Make sense! added example and mentioned that this is not needed in case the logic is not there and gave the example for the keys similar to Keycloak, but mentioned that they are tentative and can be different